lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
@shalkam Good point I overlooked the http part. Not sure if we can pass in options there.
From my understanding we can not.
@bajtos Could you help confirm here? Thanks.

And I am afraid we need to think of another strategy instead. I will put the explanation in new comment.